perf/Makefile

```diff
@@ -13,6 +19,8 @@ LDFLAGS += -L$(TARGET_OSSL_LIBRARY_PATH) -L.
 # For setting RUNPATH on built executables uncomment this:
 # LDFLAGS += -Wl,-rpath,$(TARGET_OSSL_LIBRARY_PATH)
 
+LN=/bin/ln
```

Comment:
This precludes the makefile working on non-unix like targets.  I'm not sure that's a good idea.

Reply:
the easiest hack for platforms with no symlinks we can do is just `LN=copy`.

The current `Makefile` as-is assumes unix-like environment anyway. more work is required to build tools on windows.

given we can not agree on wether implicit arguments approach is acceptable for rwlocks tests here: https://github.com/openssl/tools/pull/205 I'm going to extend run-perf.sh script we discuss here in https://github.com/openssl/tools/pull/207 so it will be able to deal with explicit arguments. I feel this approach to be easier for everyone.